src/main/scala/com/intersys/mdc/challenge/exercises/problems/Problem4.scala

```diff
@@ -45,6 +45,10 @@ case object Problem4 extends Problem {
     * Response (bad request): The request contains bad syntax or cannot be fulfilled.
     */
 
+  implicit class StringOps(string: String) {
+    def asInt: Option[Int] = scala.util.Try(Some(string.toInt)).toOption.getOrElse(None)
```

Comment:
👎 Not clean.
This makes evident a lack of understanding of `Option` usage patterns.

Reply:
This exercise could be used to help Ruben understand for comprehensions and the usage of Option patterns.